Excel to CSV

kturchin opened the pull request:
Added new module called translate_excel_to_csv - which extracts the Excel data, translates it, and converts it to a CSV file formatted as 'Title, Description, Type'. 

Several issues to refine open from this (will start working on these):

- variables which should be passed as args/have defaults ex: folder where CSV outputs should be saved
- **translation** - at the minute I'm not calling the data_check function as it was raising a lot of errors and needs to be improved anyways, I wanted to check that everything else was working fine. But without running through data_check the translation can be off especially in case of specialist terminology that might be misspelled, and worryingly with names/not translating full phrases - raising as separate issue.
- converting untranslated/translated data to the format needed for CSV is dependent on column names being spelled exactly as necessary, need to find less clunky way around it
- when the programme finishes cycling through one set of files (eg Archives) it cannot swap into Monuments even when it's reading them in folders

kturchin committed:
new module which extracts Excel data, translates it, and converts it to CSV formatted in 3 columns of Title, Description, and Type

kturchin committed:
removed old module translate_excel

razekmh commented on main.py:
```diff
@@ -20,7 +20,7 @@ def init():
         "--data_path",
         type=str,
-        required=True,
+        default=r"C:\Users\turch\OneDrive\Documents\0CAAL\KG",
```
```suggestion
        default=str(current_file_location / "data"),
```
Default data location should be relative to the location of the project. If we add the absolute data path, it is more likely to break.

razekmh committed:
refactor: update paths for input and output files

- update path of input and output to be inside the project folder
- format code

razekmh committed:
refactor: remove unsed lib

razekmh committed:
update logic checks

razekmh committed:
clean unsed code

razekmh committed:
code format

razekmh committed:
refactor: clean up combined data and set csv to quote all
